hbase-client/src/main/java/org/apache/hadoop/hbase/client/RawAsyncTableImpl.java

```diff
@@ -218,50 +220,62 @@ private static Result toResult(HBaseRpcController controller, MutateResponse res
 
   @Override
   public CompletableFuture<Result> get(Get get) {
-    return timelineConsistentRead(conn.getLocator(), tableName, get, get.getRow(),
-      RegionLocateType.CURRENT, replicaId -> get(get, replicaId), readRpcTimeoutNs,
-      conn.connConf.getPrimaryCallTimeoutNs(), retryTimer, conn.getConnectionMetrics());
+    return tracedFuture(
+      () -> timelineConsistentRead(conn.getLocator(), tableName, get, get.getRow(),
+        RegionLocateType.CURRENT, replicaId -> get(get, replicaId), readRpcTimeoutNs,
+        conn.connConf.getPrimaryCallTimeoutNs(), retryTimer, conn.getConnectionMetrics()),
+      "AsyncTable.get", tableName);
+  }
+
+  private String getSpanName(String methodName) {
```

Comment:
Typo: getSpanName not used anymore as you use AsyncTable.xxx directly.

LGTM overall.